tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy.cfg

```diff
@@ -37,3 +37,20 @@ export FSSNAP=$FS@$TESTSNAP
 export VOLSNAP=$VOL@$TESTSNAP
 export FSCLONE=$TESTPOOL/$TESTFSCLONE
 export VOLCLONE=$TESTPOOL/$TESTVOLCLONE
+if is_freebsd; then
+	export LIVELIST_MAX_ENTRIES=livelist.max_entries
+	export LIVELIST_MIN_PERCENT_SHARED=livelist.min_percent_shared
+	export LIVELIST_CONDENSE_NEW_ALLOC=livelist.condense.new_alloc
+	export LIVELIST_CONDENSE_ZTHR_CANCEL=livelist.condense.zthr_cancel
+	export LIVELIST_CONDENSE_SYNC_CANCEL=livelist.condense.sync_cancel
+	export LIVELIST_CONDENSE_ZTHR_PAUSE=livelist.condense.zthr_pause
+	export LIVELIST_CONDENSE_SYNC_PAUSE=livelist.condense.sync_pause
+else
+	export LIVELIST_MAX_ENTRIES=zfs_livelist_max_entries
+	export LIVELIST_MIN_PERCENT_SHARED=zfs_livelist_min_percent_shared
+	export LIVELIST_CONDENSE_NEW_ALLOC=zfs_livelist_condense_new_alloc
+	export LIVELIST_CONDENSE_ZTHR_CANCEL=zfs_livelist_condense_zthr_cancel
+	export LIVELIST_CONDENSE_SYNC_CANCEL=zfs_livelist_condense_sync_cancel
+	export LIVELIST_CONDENSE_ZTHR_PAUSE=zfs_livelist_condense_zthr_pause
+	export LIVELIST_CONDENSE_SYNC_PAUSE=zfs_livelist_condense_sync_pause
+fi
```

Comment:
It's unfortunate these has different names between platforms.  Are we going to need to all this kind of conditional logic for all of these tunables?

Reply:
I think there may be a few exceptions but ignoring that, I have occasionally fantasized about finding some way to automatically generate these as a tunables.cfg or somesuch.

Reply:
OK.  Well at a latter date we may want to find a way to clean this up.  That could potentially be as simple as standardizing some common names which are accepted by `set_tunable*`.  This would at least let us keep all of the translation logic in one place.  But I agree tackling that doesn't need to hold up this PR.